release.json

```diff
@@ -82,4 +82,4 @@
     "dca-1.9.0": {
         "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
     }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
As those changes are generated via script we can ignore the empty line at the end. We can consider fixing this in the script itself.